akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ReceiveBuilder.scala

```diff
@@ -71,7 +71,7 @@ class ReceiveBuilder[T] private (
    *
    * @param msg the message to compare to
    * @param handler action to apply when the message matches
-   * @return a new behavior with the specified handling appended
+   * @return this behavior builder
    */
   def onMessageEquals(msg: T, handler: Creator[Behavior[T]]): ReceiveBuilder[T] =
     withMessage(msg.getClass, Some(_.equals(msg)), _ ⇒ handler.create())
```

Comment:
CommandHandlerBuilder has `matchAny`, should we add that to the message builders too?

Reply:
Sounds good